jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/SqlDialect.java

```diff
@@ -26,13 +24,15 @@
  */
 public interface SqlDialect {
     /**
-     * @return the name that can be used to choose this dialect (user can give this name). Case insensitive.
+     * Get the dialect name
```

Comment:
```suggestion
     * Get the dialect name.
```

Reply:
Please use periods when describe classes and methods, I've unified all java docs before.